src/components/heading/heading.test.js

```diff
@@ -152,6 +152,19 @@ context("Testing Heading component", () => {
       }
     );
 
+    describe("when headingType prop is provided", () => {
+      it.each(["h1", "h2", "h3", "h4", "h5", "h6"])(
+        "should check HTML section heading element is correct when headingType is %s",
```

Comment:
```suggestion
        "should check HTML heading element is correct when headingType is %s",
```

I'm not sure that the word "section" is really adding anything helpful to this description.